### PR TITLE
Create Italy/Milan

### PR DESCRIPTION
**Pull request by BaseMax**
I am not an Italian, so any comments and requests are welcome.

### CODE REVIEW

File: doc/source/milan.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Milan, Italy
```
Comment (jbampton) [inline doc/source/milan.md:1]:
```suggestion
# 🇮🇹 Milan
```

**Review by jbampton — COMMENTED** [review]
Population figures are duplicated

**Comment by BaseMax** [comment]
thanks

**Review by jbampton — COMMENTED** [review]
Stock exchange mentioned multiple times

**Comment by BaseMax** [comment]
sorry, the wrong text was pushed. fixed.

**Comment by jbampton** [comment]
@BaseMax auto created docs ?

**Pull request closed without merging**